src/opm/input/eclipse/Schedule/Network/ExtNetwork.cpp

```diff
@@ -161,10 +197,6 @@ void ExtNetwork::update_node(Node node)
     auto branch = std::find_if(this->m_branches.begin(), this->m_branches.end(),
                                [&name](const Branch& b) { return b.uptree_node() == name || b.downtree_node() == name;});
 
-    if (branch == this->m_branches.end())
-        throw std::invalid_argument("Node: " + name + " is not referenced by any branch and would be dangling.");
-
-
     if (branch->downtree_node() == name) {
```

Comment:
Sorry that I did not spot here earlier.  Otherwise we might get a out of bound issue. 

```c++
if (branch != this->m_branches.end() && branch->downtree_node() == name) 
```